### ISSUE TITLE
`tldr tldr` shows wrong instructions

### ISSUE THREAD

**Issue by rnestler** [dbrgn/tealdeer#120]
I guess `tldr tldr` documents the official interface, not the one of tealdeer:
```
% tldr tldr

  Command-line client for tldr-pages.
  Displays simplified and community-driven man pages.
  More information: <https://tldr.sh>.

  Get typical usages of a command (hint: this is how you got here!):

      tldr command

  Show the tar tldr page for Linux:

      tldr -p linux tar

  Get help for a git subcommand:

      tldr git-checkout
```

Especially the `tldr -p linux tar` example is completely different.

**Comment by dbrgn** [comment]
Yeah, tldr just displays the downloaded tldr page. However, #106 should resolve this in case we decide to fully switch to the [CLI spec](https://github.com/tldr-pages/tldr/blob/master/CLIENT-SPECIFICATION.md#arguments).
